pkg/api/controller.go

```diff
@@ -111,10 +108,6 @@ func DumpRuntimeParams(log log.Logger) {
 	evt.Msg("runtime params")
 }
 
-func (c *Controller) GetPort() int {
```

Comment:
This was an external contribution, so that one could get at the auto-assigned port.

Reply:
commit f3faae0e09264dfa13dc7a61d38ad45ffb83bdbd
Author: Ramkumar Chinchani <45800463+rchincha@users.noreply.github.com>
Date:   Thu Sep 8 22:41:13 2022 -0700

    report listening port when chosen by kernel (#770)

    Based off of the PR by @thesayyn
    https://github.com/project-zot/zot/pull/720

    Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

Reply:
cc: @thesayyn

Reply:
Yeap. I don’t think this change is acceptable.

Reply:
https://github.com/google/go-containerregistry/pull/1586#discussion_r1128651260

Reply:
Also: https://github.com/google/go-containerregistry/pull/1586#discussion_r1128646547

Reply:
@rchincha we update the config, and by the time the controller is started the port is already chosen.
@thesayyn ok, I see there's a chance for some other process to pick the same port in between the call to GetFreePort() and the controller tries using it.

Let's close this PR?

Reply:
But the debug log still needs to be removed?

Reply:
I merged that in a separate PR.

Reply:
Then this PR can be closed. Thx @thesayyn for the review.